src/frontend-pwa/src/components/Button/Button.tsx

```diff
@@ -1,17 +1,19 @@
 import StyledButton from './button.styles';
 
 export type ButtonProps = {
-  handleClick: () => void;
+  handleClick?: () => void;
   variant: 'default' | 'primary' | 'secondary', // dictates coloring
```

Comment:
I suggest creating a `type`, maybe called `type ButtonVariants` which you can then use here. Not a huge difference but it is a good practice for creating extendable typing for components and increasing readability as things grow.

Reply:
So, for example, I would make a variant button type that has "variant, size and disabled" but not "handleClick" if I want "handleClick" to be optional? I might be misunderstanding what you mean

Reply:
oh sorry, I misunderstood what line you were referring to! I get ya

Reply:
done!